.gitignore

```diff
@@ -9,4 +9,9 @@
 *.orig
 *.*#
 *.bak
-
+/mockups
```

Comment:
Once again : Please remove these six entries . Whatever files you have in there please move them to a folder not tracked by the git repository .